projects/core/src/lib/plots/plot-list-entry/plot-list-entry.component.scss

```diff
@@ -3,7 +3,7 @@ img {
     height: auto;
 }
 
```

Comment:
strange green color in "Temporal Raster Mean Plot"  component.

![image](https://user-images.githubusercontent.com/1027102/210561859-0fea23a0-107c-4ec7-9d9e-bb8ed2d50658.png)

Reply:
Do you mean the checkbox?
It's part of the theme, but I changed it to primary :shrug: .
[f222a95](https://github.com/geo-engine/geoengine-ui/pull/555/commits/f222a95544c88185ad7c204445d73f2020b38adf)